package.json

```diff
@@ -17,11 +17,11 @@
     "start": "node node_modules/@storybook/react/bin/index -p 9001 -c .storybook",
     "build-storybook": "node node_modules/.bin/build-storybook -c .storybook -o ./storybook-static",
     "test": "npm run lint && npm run test:ci",
-    "test:ci": "npm run build-ts && cpy src/styles.scss lib && mocha --require lib/__tests__/setupEnzyme.js --recursive lib/**/*.test.js",
+    "test:watch": "jest --watch",
+    "test:ci": "jest",
```

Comment:
Was there a good reason why we weren't running the linter on CI?

Reply:
I'm not sure I understand the question but from what I understood before, there was a separate "Lint" github actions that just ran `npm run lint` and another github actions workflow that ran `npm test` which includes running `npm run lint`. It seems redundant to run it twice but maybe there was a reason for it -- but none that I could see.

Reply:
Ah ok, that makes sense!